C/other/Palindrome.c

```diff
@@ -1,51 +1,78 @@
 /*
-C Program to check whether a number is Palindrome or not
-An integer is a palindrome if the reverse of that number is equal to the original number.
-Example of Palindromes are 11,44,101,121,
+Check palindrome words using double linked list in C
+
+Input:
+aabbaa
+
+Output:
+aabbaa is palindrome
```

Comment:
Shift it to the end of the Program & Also, write Time and space complexity.
Also, Your Sample Input/Output not matches with the given Program. (Display message is not present)